packages/core-api/src/versions/2/delegates/methods.ts

```diff
@@ -17,6 +18,16 @@ const index = async request => {
     return toPagination(request, delegates, "delegate");
 };
 
+const active = async request => {
+    const delegates = await databaseService.delegates.getActiveAtHeight(request.query.height);
```

Comment:
I think `request.query.height` should default to the height of the last block.

Reply:
Yea, I'll add that. Sounds like the right thing to do.